### PR TITLE
Add gfx90a target

### PR DESCRIPTION
**Pull request by eidenyoshida**

### CODE REVIEW

**Comment by neon60** [comment]
Hi @eidenyoshida 

We also have a target list in the extra folder. Please could you update that also:
https://github.com/ROCmSoftwarePlatform/rocPRIM/blob/develop/test/extra/CMakeLists.txt#L36-L40

Thank you

**Comment by neon60** [comment]
> Hi @eidenyoshida
> 
> We also have a target list in the extra folder. Please could you update that also:
> https://github.com/ROCmSoftwarePlatform/rocPRIM/blob/develop/test/extra/CMakeLists.txt#L36-L40
> 
> Thank you

**Comment by neon60** [comment]
Sorry for this:
@neon60 closed this 39 seconds ago. Reopened.

**Review by saadrahim — APPROVED** [review]

**Comment by eidenyoshida** [comment]
Will have to revisit test/extra/CMakeLists.txt post 4.3 release

File: test/extra/CMakeLists.txt [hunk 1]
```diff
@@ -34,9 +34,9 @@ list(APPEND CMAKE_MODULE_PATH
 
 # Use target ID syntax if supported for AMDGPU_TARGETS
 if(TARGET_ID_SUPPORT)
-  set(AMDGPU_TARGETS gfx803;gfx900:xnack-;gfx906:xnack-;gfx908:xnack-;gfx1030:xnack- CACHE STRING "List of specific machine types for library to target")
+  set(AMDGPU_TARGETS set(AMDGPU_TARGETS gfx803;gfx900:xnack-;gfx906:xnack-;gfx908:xnack-;gfx1030 CACHE STRING "List of specific machine types for library to target")
```
Comment (TorreZuk) [inline test/extra/CMakeLists.txt:37]: Looks like a typo here @eidenyoshida

Comment (eidenyoshida) [reply test/extra/CMakeLists.txt:37]: Good catch, I have another PR to fix some issues I made in this one

**Pull request merged**